### ISSUE TITLE
ipfs diag net throws error

### ISSUE THREAD

**Issue by multikatt** [ipfs/go-ipfs#1513]
Using 633b66d8ec6b66a20bb8ffae8069e086bc486ecc i get the following
```
└─[$] <git:(master)> ipfs diag net
Error: Option name 'timeout' used multiple times
```
I checked https://github.com/ipfs/go-ipfs/commits/master/diagnostics/diag.go since I suspect the error comes from there and tried out commit ac7edddb94a70f5f76e23d87c3da085eb97cf34a. It works as expected in that commit.

**Comment by whyrusleeping** [comment]
whoops. thats my fault. thanks for reporting!

**Comment by jbenet** [comment]
@whyrusleeping we should fix this before cutting 0.3.6 i think

**Comment by jbenet** [comment]
should be fixed (with https://github.com/ipfs/go-ipfs/pull/1530)
